### PR TITLE
Sys 143 correct specific php versions

### PR DESCRIPTION
**Pull request by umjnega**

### CODE REVIEW

File: manifests/profile/php73.pp [hunk 1]
```diff
@@ -19,16 +19,14 @@
...
-  'php-msgpack',
+  package { ['php7.3-geoip',
+  'php7.3-igbinary',
+  'php7.3-imagick',
+  'php7.3-memcached',
+  'php7.3-msgpack',
   #  'php-pear',
-  'php-redis',
-  'php-xdebug',
-  'php-xml',
```
Comment (botimer) [inline manifests/profile/php73.pp:31]: Looks like maybe this one didn't make it over. What's the story for php-xml?

**Review by botimer — COMMENTED** [review]
Looks pretty good, but I want to be sure that php-xml is handled. For the record, I did just double check that all of these versioned packages are in packages.sury.org repository, including php7.3-xml.

File: manifests/profile/php73.pp (hunk 1 above)
Comment (umjnega) [reply manifests/profile/php73.pp:31]: are 'php-redis' and 'php-xdebug' necessary if they are specificed with 'php7.3-redis' and 'php7.3-xdebug'?   The notes on SYS-108 led me to believe they might cause the conflict?

**Review by botimer — CHANGES_REQUESTED** [review]
Cool, looks good. Let's squash this down to one commit with the issue key in the summary, and it'll be ready to go.

**Pull request merged**